### DISCUSSION TITLE
Shortcut "T" to show/hide timeline

### DISCUSSION THREAD

**Discussion by 4l3m4oO** [friction2d/friction#43] [Ideas]
I miss (again) the very useful shortcut from Enve, that shows or hide the timeline, just pressing the letter: "T"

**Comment by rodlie** [comment]
Sure, can do something about that.

**Comment by rodlie** [comment]
Fixed in https://github.com/friction2d/friction/commit/259abcc28adc38eacf2402679de0697ffdb3b798
